app/javascript/react/screens/App/Overview/components/ModalWizard/ModalWizard.js

```diff
@@ -3,14 +3,18 @@ import PropTypes from 'prop-types';
 import { noop, Modal, Wizard, Icon, Button } from 'patternfly-react';
 import { connect } from 'react-redux';
 
+// TODO we should lift this application-specific stuff out of this generalized component file.
 const reduxFormMap = {
   [__('Infrastructure Mapping Wizard')]: [
     'mappingWizardGeneralStep',
     'mappingWizardClustersStep'
   ],
-  [__('Migration Plan Wizard')]: []
+  [__('Migration Plan Wizard')]: ['planWizardGeneralStep']
```

Comment:
@mturley Add `planWizardCSVStep` here in `reduxFormMap` for Next to work.
Related to https://github.com/priley86/miq_v2v_ui_plugin/pull/64